examples/codes/filter/dubbo/install/Makefile

```diff
@@ -137,4 +137,4 @@ shell:
 
 proxyv2:
 	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o ./build/mosn -ldflags "-X mosn.io/mosn/pkg/types.IstioVersion=${ISTIO_VERSION}" ../../../../../cmd/mosn/main
-	docker build -t symcn.tencentcloudcr.com/symcn/proxyv2:${ISTIO_VERSION} -f ./build/Dockerfile .
+	docker build -t registry.cn-hangzhou.aliyuncs.com/champly/proxyv2:${ISTIO_VERSION} -f ./build/Dockerfile .
```

Comment:
It's better to use this url: https://hub.docker.com/repository/docker/mosnio/proxyv2

Reply:
replace with docker.io/mosnio